qiskit/extensions/standard/rzz.py

```diff
@@ -54,15 +54,9 @@ def reapply(self, circ):
         self._modifiers(circ.rzz(self.param[0], self.qargs[0], self.qargs[1]))
 
 
+@_control_target_gate
 def rzz(self, theta, ctl, tgt):
```

Comment:
rzz is not a `control_target_gate`. The naming of the operands as `ctl`/`tgt` is a result of copy-paste and should be changed.
It is just a regular 2-qubit gate. In particular, that means `len(ctrl)` should be equal to `len(tgt)`.
Maybe we can have a separate decorator for this type of gate (would also include the swap gate).